docs/changes.rst

```diff
@@ -212,6 +212,11 @@ Released: not yet
 * Improvements in the log messages of the `MOFCompiler` class.
   (related to issue #2586)
 
+* Added a `close()` method to `pywbem.WBEMConnection` that closes the underlying
+  session of the 'requests' package. This avoids the ResourceWarning
+  'unclosed socket' that the 'requests' package issued so far when the Python
+  process terminates. (see issue #2591)
+
```

Comment:
Should we also note that the class is context-aware??

Reply:
Good point. Added that to the change log. DONE